Update Debian Repo to 2.*

crpb opened the pull request:
The release in Debian throws an error and it seems the maintainer hasn't got the time.

ref: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1076361
ref: #34

novalis111 commented:
Time to push this into the repos 👍 This warning is annoying.

crpb committed:
add gbp.conf

crpb committed:
New upstream version 2.36.0

crpb committed:
Update upstream source from tag 'upstream/2.36.0'

Update to upstream version '2.36.0'
with Debian dir f96e8fda2dec1609e29c76b108b979589d708d3c

sblondon commented:
@punitagrawal Trixie has been released so the freeze is over. Do you plan to update the package ?

crpb committed:
New upstream version 2.36.0

crpb committed:
Update upstream source from tag 'upstream/2.36.0'

Update to upstream version '2.36.0'
with Debian dir c976b514c6749ef08f2cc232850ec79c825e186d

crpb committed:
New upstream version 2.36.0

crpb committed:
New upstream version 2.36.0

crpb committed:
Update upstream source from tag 'upstream/2.36.0'

Update to upstream version '2.36.0'
with Debian dir c976b514c6749ef08f2cc232850ec79c825e186d

crpb committed:
New upstream version 2.37.1

crpb committed:
Update upstream source from tag 'upstream/2.37.1'

Update to upstream version '2.37.1'
with Debian dir c976b514c6749ef08f2cc232850ec79c825e186d

crpb committed:
d/changelog - new upstream release 2.37.1

punitagrawal commented:
Hello, yes I will be updating the packages in the coming days.
Sooner rather than later.

On Sun, Aug 17, 2025 at 12:16 PM sblondon ***@***.***> wrote:

> *sblondon* left a comment (direnv/direnv-deb#35)
> <https://github.com/direnv/direnv-deb/pull/35#issuecomment-3194317220>
>
> @punitagrawal <https://github.com/punitagrawal> Trixie has been released
> so the freeze is over. Do you plan to update the package ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/direnv/direnv-deb/pull/35#issuecomment-3194317220>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAGBBSTABJRJWZKQ64SWQ633OBQATAVCNFSM6AAAAABZC2VXAKVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTCOJUGMYTOMRSGA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
